images/tidb-backup-manager/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM pingcap/tidb-enterprise-tools:latest
 
 ARG VERSION=v1.48.0
```

Comment:
Please update rclone to latest stable version v1.51.0

Reply:
done

Reply:
done